### PR TITLE
fix: upgrade to middy@1.0.0

### PR DESCRIPTION
**Pull request by steelerc69**
Hoping you are open to a PR for the upgrade to middy 1.0.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +36,10 @@
     "jsonwebtoken": "^8.4.0"
   },
   "peerDependencies": {
-    "middy": "^0.22.0"
+    "@middy/core": "^1.0.0",
+    "@middy/http-error-handler": "^1.0.0",
+    "@middy/http-header-normalizer": "^1.0.0",
+    "@types/aws-lambda": "^8.10.51"
```
Comment (dbartholomae) [inline package.json:42]: I don't think these (except for @middy/core) should be peerDependencies as they are not required for using the package, only for testing it. Please move to devDependencies instead.

Comment (dbartholomae) [reply package.json:42]: They shouldn't be in dependencies either.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "middy-middleware-jwt-auth",
-  "version": "3.0.4",
+  "version": "4.0.0",
```
Comment (dbartholomae) [inline package.json:3]: Version number and changelog are managed by semantic release and will be added automatically once the PR is merged. Please remove these changes from the PR.
For this to work, the commit message is important: the commit message text will be used for the change log. Please add the following at the end of the commit message to ensure that the major version is bumped correctly and the changelog has an explanation:
```
BREAKING CHANGE: As of this version this middleware only supports middy@1.0.0 and upwards. See https://github.com/middyjs/middy/blob/master/UPGRADE.md for middy upgrade instructions.
```

**Comment by steelerc69** [comment]
Hi not used the commitlint before, so hopefully the commit message is as you wanted/need.  I believe the remainder of the changes are completed.

**Comment by dbartholomae** [comment]
Rebasing into one commit is needed, otherwise the message will appear twice in the changelog. I'll quickly fix it. Thanks for the help! :)

**Comment by steelerc69** [comment]
thanks for the quick turn around on this...have a great day

**Comment by dbartholomae** [comment]
Superseded by #79

**Pull request closed without merging**